src/plugins/banner/public/plugin.test.ts

```diff
@@ -31,23 +34,27 @@ describe('BannerPlugin', () => {
   });
 
   describe('start', () => {
-    test('calls renderBanner with http client', async () => {
+    test('sets global banner in chrome service', async () => {
       const coreStart = coreMock.createStart();
+      // Mock the setGlobalBanner method
+      coreStart.chrome.setGlobalBanner = setGlobalBannerMock;
+
       const plugin = new BannerPlugin();
       const start = await plugin.start(coreStart);
 
       expect(start).toEqual({});
-      expect(renderBannerMock).toHaveBeenCalledTimes(1);
-      expect(renderBannerMock).toHaveBeenCalledWith(coreStart.http);
+      expect(setGlobalBannerMock).toHaveBeenCalledTimes(1);
+      expect(setGlobalBannerMock.mock.calls[0][0]).toHaveProperty('component');
     });
   });
 
   describe('stop', () => {
-    test('calls unmountBanner', () => {
+    test('banner is automatically unmounted when plugin stops', () => {
       const plugin = new BannerPlugin();
       plugin.stop();
 
-      expect(unmountBannerMock).toHaveBeenCalledTimes(1);
+      // No explicit assertion needed as the banner is automatically unmounted
+      // when the plugin is stopped, as noted in the implementation comment
```

Comment:
remove?

Reply:
Also curious, what is purpose of this test if its not asserting anything? That nothing is thrown?